pkg/e2e/framework/common.go

```diff
@@ -41,23 +39,9 @@ const (
 
 // RandomString returns a random 6 character string.
 func RandomString() string {
-	randID := string(uuid.NewUUID())
+	randID := "e2e-" + string(uuid.NewUUID())
```

Comment:
can we please make sure we follow this pattern https://github.com/openshift/cluster-api-actuator-pkg/pull/131/files#diff-798770aa2c88fce8a6789bb252bcb627R394-R397